lms/djangoapps/grades/api/tests/test_views.py

```diff
@@ -68,34 +68,35 @@ def setUpClass(cls):
         super(CurrentGradeViewTest, cls).setUpClass()
 
         cls.course = CourseFactory.create(display_name='test course', run="Testing_course")
+        with cls.store.bulk_operations(cls.course.id):
```

Comment:
**Note to reviewers:**  Moving the creation of the test course into a bulk_operation significantly improves the setup time for these tests (since a publish-signal isn't sent for each of the 75+ blocks created).